src/containers/adminDashboard/index.tsx

```diff
@@ -1,84 +1,107 @@
-import { DARK_GREEN } from '../../utils/colors';
-import ProtectedClient from '../../api/protectedApiClient';
-import {
-  AddSiteRequest,
-  UpdateSiteRequest,
-} from '../../components/forms/ducks/types';
-import UpdateSiteForm from '../../components/forms/updateSiteForm';
-import EditSiteForm from '../../components/forms/editSiteForm';
-import SelectorMapDisplay from '../../components/mapComponents/mapDisplays/selectorMapDisplay';
 import { MapGeoDataReducerState } from '../../components/mapComponents/ducks/types';
-import { MapContainer, Block, Flex } from '../../components/themedComponents';
-import useWindowDimensions, {
-  WindowTypes,
-} from '../../components/windowDimensions';
-import { getMapGeoData } from '../../components/mapComponents/ducks/thunks';
+import { Flex } from '../../components/themedComponents';
 import SignupForm from '../../components/forms/signupForm';
 import { SignupFormValues } from '../../components/forms/ducks/types';
 import ProtectedApiClient from '../../api/protectedApiClient';
 import { AppError } from '../../auth/axios';
 import { getErrorMessage } from '../../utils/stringFormat';
 import { SubmitButton } from '../../components/themedComponents';
-import { round } from 'lodash';
-import { LAT_LNG_PRECISION } from '../../components/forms/constants';
-import { MapTypes } from '../../context/types';
-import { MapTypeContext } from '../../context/mapTypeContext';
+import {
+  BarChartOutlined,
+  FileAddOutlined,
+  MailOutlined,
+  PlusOutlined,
+  RocketFilled,
+  SettingFilled,
+} from '@ant-design/icons';
+import Image1 from '../../assets/images/bkg1.png';
+import Image2 from '../../assets/images/bkg2.png';
+import Image4 from '../../assets/images/bkg4.png';
+import { Routes } from '../../App';
 
 const AdminContentContainer = styled.div`
   width: 80vw;
   margin: 8vh auto auto;
 `;
 
-const DashboardContent = styled.div`
-  width: 450px;
+const ImageCard = styled.div`
+  background-color: rgba(0, 0, 0, 0.25);
+  background-blend-mode: darken;
+  border-radius: 10px;
+  width: 230px;
+  height: 230px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  color: white;
+  font-size: 16pt;
+  font-weight: bold;
+  transition: background-color 0.2s;
+
+  &:hover {
+    background-color: rgba(0, 0, 0, 0.5);
+  }
 `;
 
 const AdminDivider = styled(Divider)`
   margin-top: 10px;
   margin-bottom: 20px;
 `;
 
-const SectionHeader = styled(Typography.Text)`
-  font-weight: bold;
-  font-size: 20px;
-  color: ${DARK_GREEN};
-`;
+interface ImageLinkCardProps {
+  href: string;
+  image: string;
+}
 
-const MarginBottomRow = styled(Row)`
-  margin-bottom: 30px;
-`;
+const ImageLinkCard: React.FC<ImageLinkCardProps> = ({
+  href,
+  image,
+  children,
+}) => (
+  <a href={href}>
+    <ImageCard style={{ backgroundImage: `url(${image})` }}>
+      {children}
+    </ImageCard>
+  </a>
+);
+
+const ICON_SIZE = 40;
 
 interface AdminDashboardProps {
```

Comment:
⛏️ this should be an unused interface that can be deleted (not sure why the linter didn't catch this tho)